localstack-core/localstack/services/opensearch/packages.py

```diff
@@ -216,6 +226,15 @@ class ElasticsearchPackageInstaller(PackageInstaller):
     def __init__(self, version: str):
         super().__init__("elasticsearch", version)
 
+        # JRE version to use
+        # ElasticSearch has the widest compatibility with Java 8
+        # See: https://www.elastic.co/support/matrix
+        self.java_version = "8"
```

Comment:
Actually, ElasticSearch 5.1 until 6.4 ([which we support](https://github.com/localstack/localstack/blob/1a897f99b5943bc8985abe6b88d682ada8c96f14/localstack-core/localstack/services/opensearch/versions.py#L40-L47)) do not support Java 11. We do not test older versions which caused this to go unnoticed.

Reply:
Okay, that's surprising, because that would mean that it won't work at all right now. So this would address an issue in our current code which I haven't been made aware of yet.
Have you tested that this currently doesn't work and will be fixed with this change?

Reply:
We now use the bundled JRE with 07e9675270476a674105bbc9f977374365cc3f19 so this is moot.